cordapp-example/contracts-kotlin/src/main/kotlin/com/example/state/CashState.kt

```diff
@@ -17,7 +17,7 @@ data class CashState(val value: Long, val creator: AbstractParty, override val o
     override fun generateMappedObject(schema: MappedSchema): PersistentState {
         return when (schema) {
             is CashSchemaV1 -> CashSchemaV1.PersistentCash(
-                    this.creator.nameOrNull().toString(),
+                    this.creator.owningKey.toString(),
```

Comment:
By the way, is `AbstractParty` accepted as a "primitive" by the schema?

Reply:
sorry, im not follow... please explain more.

Reply:
Can `CashSchemaV1.PersistentCash` take an `AbstractParty` as its first parameter?

Reply:
sure, done